sharding/data_modeling.rst

```diff
@@ -10,6 +10,31 @@ Typically, you should choose that column as the distribution column which is the
 .. note::
   Citus also allows joining on non-distribution key columns by dynamically repartitioning the tables for the query. Still, joins on non-distribution keys require shuffling data across the cluster and therefore aren’t as efficient as joins on distribution keys.
 
-The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. In the former all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+Use Cases and Tradeoffs
+-----------------------
 
-The other common option, realtime analytics, distributes by another column (such as user id). The queries in this scenario typically request aggregates information from multiple shards. This permits query parallelism but restricts some of the SQL features available, due to the constraints of being a distributed system.
+The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. Both cases involve tradeoffs.
+
+Multi-Tenant
+~~~~~~~~~~~~
+
+* ✔ SQL Coverage
+* ✔ Ease of Migration
+* ✘ Less Parallelism
+* ✘ Less effective for large (or few) tenants
+
+In the multi-tenant use case all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
```

Comment:
So I think this content is good but should probably be expanded a bit. We want to establish what multi-tenancy is, and how it would work in Citus. So basically, what does 'tenant id' mean? That may not be immediately obvious. We should also discuss the concept of co-location, and thus its necessary that *every* table is partitioned by the same tenant-id column. The blog post has some good content we can use: https://www.citusdata.com/blog/2016/08/10/sharding-for-a-multi-tenant-app-with-postgres/. Providing example use-cases like the blog-post does would help in allowing a user to identify.
It probably also makes sense to link to that post.